uint16 servo1
uint16 servo2
uint16 servo3
uint16 servo4
uint16 servo5
uint16 servo6
uint16 servo7
